docs/src/reference/gremlin-variants.asciidoc

```diff
@@ -100,7 +100,7 @@ Gremlin-Python users will typically make use of the following classes.
 
 [source,python]
 >>> from gremlin_python import statics
->>> from gremlin_python.structure.graph import Graph
+>>> from gremlin_python.structure.graph import traversal
```

Comment:
To match java/C#/javascript, I think we should expose it under `process` not `structure`.